tests/Unit/CliSamplingTest.php

```diff
@@ -175,4 +185,83 @@ public function test_it_applies_individual_sample_rates_to_scheduled_tasks(): vo
         $this->assertSame(100, $writes);
         $this->assertCount(0, $this->core->ingest->buffer);
     }
+
+    #[DataProvider('vendorCommands')]
+    public function test_it_samples_vendor_commands_separately(string $command): void
```

Comment:
"Separately" was a bit unclear to me. What do you think about this?

```suggestion
    public function test_it_does_not_sample_vendor_commands(string $command): void
```